plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -84,26 +84,24 @@ class RsyncConf:
 
 
 @dataclass
-class PluginMandatoryInstallConf:
-    hydra_version: str = hydra.__version__
-    ray_version: str = ray.__version__
-    cloudpickle_version: str = cloudpickle.__version__
-    omegaconf_version: str = omegaconf.__version__
-    pickle5_version: str = pkg_resources.get_distribution("pickle5").version
-    hydra_ray_launcher_version: str = pkg_resources.get_distribution(
-        "hydra_ray_launcher"
-    ).version
+class MandatoryInstallConf:
+    pip_packages: Dict[str, str] = field(
+        default_factory=lambda: {
+            "hydra_core": hydra.__version__,
+            "ray": ray.__version__,
+            "cloudpickle": cloudpickle.__version__,
+            "omegaconf": omegaconf.__version__,
+            "pickle5": pkg_resources.get_distribution("pickle5").version,
+            "hydra_ray_launcher": pkg_resources.get_distribution(
+                "hydra_ray_launcher"
+            ).version,
```

Comment:
Ok, now I understand where the specific versions are coming from.
You can remove OmegaConf, installing Hydra will pull the proper version in.

Reply:
OmegaConf is a key lib, i think it's worth it to pin the version so it's less likely to cause issue in the remote environment. (it might also be hard to debug when a version mismatch happens)